test/GovernorCountingFractional.t.sol

```diff
@@ -332,7 +332,15 @@ contract GovernorCountingFractionalTest is Test {
       assertEq(uint8(status), uint8(IGovernor.ProposalState.Defeated));
 
       Proposal memory _rawProposalInfo = _getSimpleProposal();
-      vm.expectRevert(bytes("Governor: proposal not successful"));
+      vm.expectRevert(
+        abi.encodeWithSelector(
+          IGovernor.GovernorUnexpectedProposalState.selector,
+          _proposalId,
+          status,
+          bytes32(1 << uint8(IGovernor.ProposalState.Succeeded))
```

Comment:
From this line https://github.com/OpenZeppelin/openzeppelin-contracts/blob/920225a1c7e6e749ba23b10ed076b04f454290e8/contracts/governance/Governor.sol#L403